charts/linkerd2/templates/web-rbac.yaml

```diff
@@ -102,6 +102,36 @@ subjects:
   name: linkerd-web
   namespace: {{.Values.Namespace}}
 ---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
if the web is not including the injector dry run check, it should no longer need these RBAC permissions, correct?

Reply:
It's included at the moment by default in both the CLI and the dashboard (and can be disabled in the CLI by the user using the new flag). But it's easy to remove now if you prefer this one off in the dashboard.

Reply:
yeah, I think we should remove it from the dashboard in order to avoid needing to give "create" permissions to web

Reply:
Done 👍